tracer/src/Datadog.Trace/Telemetry/Metrics/Count.cs

```diff
@@ -115,6 +115,16 @@ internal enum Count
     /// The number of requests sent to the api endpoint in the agent that errored, tagged by the error type (e.g. `type:timeout`, `type:network`, `type:status_code`)
     /// </summary>
     [TelemetryMetric<MetricTags.ApiError>("stats_api.errors")] StatsApiErrors,
+
+    /// <summary>
+    /// The number of times a Datadog configuration is set while a corresponding OpenTelemetry configuration is set.
+    /// </summary>
+    [TelemetryMetric<MetricTags.DatadogConfiguration, MetricTags.OpenTelemetryConfiguration>("otel.env.hiding", isCommon: true, NS.Tracer)] OpenTelemetryConfigHiddenByDatadogConfig,
```

Comment:
Have these been allow-listed in the telemetry intake?

Reply:
Not yet...we are tracking that work though